docs/UserGuide.md

```diff
@@ -913,11 +1063,77 @@ Command Words Accepted: `list`, `ls` (case-insensitive)
 > ```
 
 #### Clear Transactions
+
 Clears all transactions in UniCa$h.
 
 Command: `clear_transactions`
 
-Command Words Accepted: `clear_transactions` (case-insensitive)
+Command Words Accepted: `clear_transactions` only (case-insensitive)
+
+<div class="callout callout-important" markdown="span" style="margin-bottom: 20px;">
+This command irreversibly deletes all transactions in UniCa$h. Thus, as an added layer
+of safety, the command has no short form alternatives and is intentionally lengthy. Additionally,
+it cannot be followed with any arguments, options, or non-whitespace text.
+</div>
+
+##### Successful Execution
+
+###### Example 1
+
+> **Case**: Command entered with correct format
+>
+> **Input**: `clear_transactions`
+>
+> **Output**:
+> ```
+> All transactions have been cleared!
+> ```
+>
+> <img src="images/unicash/command-outputs/clearTransactions/clear.png" width="1000" />
+
+###### Example 2
+
+> **Case**: Command entered with acceptable alternate format
+>
+> **Input**: `CLEAR_TRANSACTIONS`
+>
+> **Output**:
+> ```
+> All transactions have been cleared!
+> ```
+> 
+>  _(UI Output Same as above)_
+
+##### Failed Execution
```

Comment:
Think both cases can also be reduced down to mentioning that any additional argument or prefix will be flagged as an error instead. Then the note can be a callout to keep this section clean

Reply:
Done! Added a callout at the bottom instead